common.yml

```diff
@@ -823,2402 +822,2612 @@ timeout:
+
+  - name: rhel72-s390x
+    display_name: ZAP s390x RHEL 7.2
+    run_on:
+      - rhel72-zseries
+    stepback: false
+    expansions:
+      <<:
+        [
+          *mongod_ssl_startup_args,
+          *mongo_ssl_startup_args,
+          *mongod_tls_startup_args,
+          *mongo_tls_startup_args,
+        ]
+      mongo_os: "rhel72"
+      mongo_edition: "enterprise"
+      mongo_arch: "s390x"
+      smoke_use_ssl: --use-ssl
+      resmoke_use_ssl: _ssl
+      smoke_use_tls: --use-tls
+      resmoke_use_tls: _tls
+      excludes: requires_mmap_available,requires_mongo_24,requires_mongo_26,requires_mongo_30
+      resmoke_args: -j 2
+      edition: enterprise
+      run_kinit: true
+    tasks:
+      - name: "unit"
+      - name: ".4.4"
+      - name: ".5.0"
+      # We did not do a release of 5.3 for RHEL 7.2 on s390x. There should be a
+      # 6.x release in the future, but there is no 6.0.
+      - name: ".5.1"
+      - name: ".5.2"
+      - name: ".latest"
+      - name: ".kerberos"
+      - name: "dist"
+      - name: "sign"
+        run_on: rhel80-small
+      - name: "push"
+        run_on: rhel80-small
+
+  - name: rhel83-s390x
+    display_name: ZAP s390x RHEL 8.3
+    run_on:
+      - rhel83-zseries
+    stepback: false
+    expansions:
+      <<:
+        [
+          *mongod_ssl_startup_args,
+          *mongo_ssl_startup_args,
+          *mongod_tls_startup_args,
+          *mongo_tls_startup_args,
+        ]
+      mongo_os: "rhel83"
+      mongo_edition: "enterprise"
+      mongo_arch: "s390x"
+      smoke_use_ssl: --use-ssl
+      resmoke_use_ssl: _ssl
+      smoke_use_tls: --use-tls
+      resmoke_use_tls: _tls
+      excludes: requires_mmap_available,requires_mongo_24,requires_mongo_26,requires_mongo_30
+      resmoke_args: -j 2
+      edition: enterprise
+      run_kinit: true
+    tasks:
+      - name: "unit"
+      # We only have 5.0.x and 6+ releases for RHEL 8.x on s390x
+      - name: ".5.0"
+      #  - name: ".latest"
+      - name: ".kerberos"
+      - name: "dist"
+      - name: "sign"
+        run_on: rhel80-small
+      - name: "push"
+        run_on: rhel80-small
+
+  #######################################
+  #     Experimental Buildvariants      #
+  #######################################
+
+  - name: ubuntu-race
+    stepback: false
+    display_name: ~ Race Detector Ubuntu 18.04
+    run_on:
+      - ubuntu1804-small
+    expansions:
+      <<:
+        [
+          *mongod_ssl_startup_args,
+          *mongo_ssl_startup_args,
+          *mongod_tls_startup_args,
+          *mongo_tls_startup_args,
+        ]
+      mongo_os: "ubuntu1804"
+      mongo_edition: "enterprise"
+      args: "-buildmode=default -race"
+      excludes: requires_large_ram
+      resmoke_use_tls: _tls
+      resmoke_use_ssl: _ssl
+      USE_SSL: "true"
+    tasks:
+      - name: "unit"
+      - name: ".3.6"
+      - name: ".4.0"
+      - name: ".4.2"
+      - name: ".4.4"
+
+github_pr_aliases:
```

Comment:
Could we add a comment saying that this is generated and how it was generated? That way people won't accidentally manually edit it (I hope).

Reply:
Yes, good call! I added this.